net-im/signal-desktop-bin/signal-desktop-bin-1.0.38.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="${PN/-bin/}"
+
+inherit eutils gnome2-utils unpacker xdg-utils
+
+DESCRIPTION="Signal Desktop"
+HOMEPAGE="https://signal.org/ https://github.com/WhisperSystems/Signal-Desktop"
+SRC_URI="https://updates.signal.org/desktop/apt/pool/main/s/${MY_PN}/${MY_PN}_${PV}_amd64.deb"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RESTRICT="bindist mirror"
+
+DEPEND="
+	gnome-base/gconf:2
+	dev-libs/libappindicator:3
+	dev-libs/nss
+	x11-libs/libXtst
+	"
```

Comment:
if this is a binary package, you don't need `DEPEND`

Reply:
Changed.